src/cc65/declare.c

```diff
@@ -2079,8 +2092,10 @@ static unsigned ParseStructInit (Type* T, int AllowFlexibleMembers)
 
             /* Account for the data and output it if we have a full word */
             SI.ValBits += Entry->V.B.BitWidth;
-            CHECK (SI.ValBits <= INT_BITS);
-            if (SI.ValBits == INT_BITS) {
+            /* Make sure unsigned is bit enough to hold the value, 30 bits. */
```

Comment:
That comment should say,"... _big_ enough ...".

Why is the limit 30 instead of 32?

Reply:
Fixed and expanded comment.